### DISCUSSION TITLE
Flixel Newb - HTML5 receive events.js:72

### DISCUSSION THREAD

**Discussion by ikanchev1** [HaxeFlixel/flixel#2382] [Q&A]
Hi all,

I am quite new to HaxeFlixel coding. I am following the documentation and i am currently trying to run my first project according to this:
https://haxeflixel.com/documentation/creating-a-new-project/

I should mention i am using Windows 10.

If i use Neko, HashLink the app works - starts new window and i see the text. If i try to use HTML5 however and run i get this error. From what i read online it seems to be common issue, something hanging the process. But which process is this? Please advise:
events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: listen EADDRINUSE
    at errnoException (net.js:904:11)
    at Server._listen2 (net.js:1042:14)
    at listen (net.js:1064:10)
    at net.js:1146:9
    at dns.js:72:18
    at process._tickCallback (node.js:419:13)
    at Function.Module.runMain (module.js:499:11)
    at startup (node.js:119:16)
    at node.js:906:3
The terminal process "C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -Command lime test html5 -debug --connect 6000" terminated with exit code: 1.

**Comment by RichardBray** [comment]
So HaxeFlixel games run on port 8080 by default and according to that error message it looks like you have something already running on that port. If you have nodejs installed this command should kill whatever is running on that port:
```
npx kill-port 8080
```
